mmcv/cnn/bricks/conv_module.py

```diff
@@ -105,7 +105,11 @@ def __init__(self,
         self.with_bias = bias
 
         if self.with_norm and self.with_bias:
-            warnings.warn('ConvModule has norm and bias at the same time')
+            check_content = NORM_LAYERS.get(self.norm_cfg.get('type'))
```

Comment:
maybe we can check here to simplify the code
https://github.com/open-mmlab/mmcv/blob/ac0d8398bfd5d513848f37f20bb75e5cd25bc5c3/mmcv/cnn/bricks/conv_module.py#L148

For example
```python
        # build normalization layers
        if self.with_norm:
            # norm layer is after conv layer
            if order.index('norm') > order.index('conv'):
                norm_channels = out_channels
            else:
                norm_channels = in_channels
            self.norm_name, norm = build_norm_layer(norm_cfg, norm_channels)
            if self.with_bias and issubclass(norm, check_range):
                ...
            self.add_module(self.norm_name, norm)
```

Reply:
btw, do I need to remove the ruby source for windows os from CONTRIBUTING.md ?

Reply:
yet, you can create another PR?

Reply:
Can `check_range` be renamed to `batch_norm_list`?

Reply:
> yet, you can create another PR?

I don't think it worth another PR, hahaha. Thanks  you very much, the pre-commit tool saves me a lot of time.

Reply:
haha, thanks for your contribution.